src/Plugin/EnhancedButtonInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\enhanced_button_link\Plugin;
```

Comment:
Move EnhancedButtonInterface to namespace Drupal\enhanced_button_link.
File EnhancedButtonInterface.php should be placed to src directory (level up)

Reply:
done